tests/sentry/api/endpoints/test_organization_member_details.py

```diff
@@ -19,344 +19,248 @@
+        )
+
+        response = self.get_success_response(self.organization.slug, member_om.id)
+        assert team.slug in response.data["teams"]
+
+
+class UpdateOrganizationMemberTest(OrganizationMemberTestBase):
+    method = "put"
+
+    def test_invalid_id(self):
+        self.get_error_response(self.organization.slug, "trash", reinvite=1, status_code=404)
+
+    @patch("sentry.models.OrganizationMember.send_invite_email")
+    def test_reinvite_pending_member(self, mock_send_invite_email):
+        member_om = self.create_member(
+            organization=self.organization, email="foo@example.com", role="member"
+        )
+
+        self.get_success_response(self.organization.slug, member_om.id, reinvite=1)
         mock_send_invite_email.assert_called_once_with()
 
     @patch("sentry.utils.ratelimits.for_organization_member_invite")
     @patch("sentry.models.OrganizationMember.send_invite_email")
     def test_rate_limited(self, mock_send_invite_email, mock_rate_limit):
         mock_rate_limit.return_value = True
 
-        organization = self.create_organization(name="foo", owner=self.user)
         member_om = self.create_member(
-            organization=organization, email="foo@example.com", role="member"
+            organization=self.organization, email="foo@example.com", role="member"
         )
 
-        path = reverse(
-            "sentry-api-0-organization-member-details", args=[organization.slug, member_om.id]
-        )
-
-        self.login_as(self.user)
+        self.get_error_response(self.organization.slug, member_om.id, reinvite=1, status_code=429)
 
-        resp = self.client.put(path, data={"reinvite": 1})
-        assert resp.status_code == 429
         assert not mock_send_invite_email.mock_calls
 
     @patch("sentry.models.OrganizationMember.send_invite_email")
     def test_member_cannot_regenerate_pending_invite(self, mock_send_invite_email):
-        self.login_as(user=self.user)
-
-        organization = self.create_organization(name="foo", owner=self.user)
         member_om = self.create_member(
-            organization=organization, email="foo@example.com", role="member"
+            organization=self.organization, email="foo@example.com", role="member"
         )
         old_invite = member_om.get_invite_link()
 
         member = self.create_user("baz@example.com")
-        self.create_member(organization=organization, user=member, role="member")
-
-        path = reverse(
-            "sentry-api-0-organization-member-details", args=[organization.slug, member_om.id]
-        )
-
+        self.create_member(organization=self.organization, user=member, role="member")
         self.login_as(member)
 
-        resp = self.client.put(path, data={"reinvite": 1, "regenerate": 1})
-
-        assert resp.status_code == 403
+        self.get_error_response(
+            self.organization.slug, member_om.id, reinvite=1, regenerate=1, status_code=403
```

Comment:
this is inconsequential but I'm surprised you didn't put reinvite and regen into a data object

Reply:
In general I set `data` on a separate line if it has camelcased keys or if it's too large to inline. I think you're right that it's worth always doing for consistency.